docs/spaces/securing-spaces.asciidoc

```diff
@@ -4,4 +4,6 @@
 
 With security enabled, you can control who has access to specific spaces. You can manage access in **Management > Roles**.
 
-image::spaces/images/securing-spaces.png["Securing spaces"]
\ No newline at end of file
+image::spaces/images/securing-spaces.png["Securing spaces"]
+
+Note that kibana ships with a built-in role called ```kibana_user``` which grants the **all** privilege as minimum access to all spaces. In order to restrict a user to only a subset of spaces, remember to not give the ```kibana_user``` role but but instead to create a custom scheme where new roles grant both Index Privileges **manage, read, index, delete** to the pattern ```.kibana*``` and the intended access to the correct subset of spaces.
```

Comment:
We actually don't want to be creating roles with direct index access anymore, and should be recommending using the Kibana Privileges.

Reply:
@kobelb  The built in kibana roles grant access to all spaces, so it's a circular argument for the person who wants to restrict some spaces for some users.  I'm not sure what to suggest.

Reply:
This is the section of the role page, which we refer to as the "Kibana privileges":

<img width="990" alt="screen shot 2018-12-06 at 10 38 05 am" src="https://user-images.githubusercontent.com/627123/49604589-17aae180-f943-11e8-9a62-ebc0c0505690.png">